app/src/main/res/menu/menu_player.xml

```diff
@@ -23,4 +23,8 @@
         android:id="@+id/action_set_alarm"
         android:icon="@drawable/ic_query_builder_black_24dp"
         android:title="@string/action_alarm" />
+    <item
+        android:id="@+id/action_delete_stream_history"
+        android:icon="@drawable/ic_delete_black_24dp"
+        android:title="@string/nav_item_delete_stream_history"/>
```

Comment:
I think `action_delete_stream_history` would be more appropriate

Reply:
what do you mean?

Reply:
I mean to `nav_item_delete_stream_history` -> `action_delete_stream_history` since this is action and not text for navigation